admin/class-admin-init.php

```diff
@@ -105,8 +105,7 @@ public function tagline_notice() {
 				return;
 			}
 
-			$current_url = ( is_ssl() ? 'https://' : 'http://' );
-			$current_url .= sanitize_text_field( $_SERVER['SERVER_NAME'] ) . sanitize_text_field( $_SERVER['REQUEST_URI'] );
+			$current_url = admin_url( sanitize_text_field( filter_input( INPUT_SERVER, 'REQUEST_URI' ) ) );
```

Comment:
`filter_input()` is not used in WP core, as it may not be enabled on some installs. WPSEO would need to have this as a requirement, or provide a fallback if the function doesn't exist (which it may do, as I may have highlighted this before, but better to raise it than ignore it).